apps/src/lab2/Lab2MetricsReporter.ts

```diff
@@ -12,12 +12,16 @@ interface ReportingProperties {
 }
 
 /**
- * Singleton metrics reporter for Lab2 labs.
+ * Metrics reporter for labs.
  */
-class Lab2MetricsReporter {
+export class LabMetricsReporter {
```

Comment:
Curious why you're not renaming file.

Reply:
To avoid a bunch of changes in Lab2 files 🙂 the next step in this process will be to switch over Lab2 code to use the reporter via the Lab2Registry, and then after that, we can rename this file and move it out of the Lab2 directory. Wanted to avoid inflating this PR too much.